docs/tutorials/datasets/det.py

```diff
@@ -0,0 +1,38 @@
+"""Prepare ILSVRC 2015DET dataset
+=========================================
+
+`ILSVRC DET dataset <http://image-net.org/>`_ is
+a dataset for object localization/detection and image/scene classification task.
+It contains two thousands full labeled categories for detection and one thousands categories for 1000 categories
```

Comment:
What do you mean by "one thousands categories for 1000 categories"? Maybe a typo. I think you are trying to say 1000 categories for classification.